pom.xml

```diff
@@ -239,31 +239,31 @@
         <dependency>
             <groupId>org.codehaus.plexus</groupId>
             <artifactId>plexus-archiver</artifactId>
-            <version>3.4</version>
+            <version>4.2.2</version>
         </dependency>
         <dependency>
             <groupId>commons-io</groupId>
             <artifactId>commons-io</artifactId>
-            <version>2.6</version>
+            <version>2.7</version>
             <scope>provided</scope>
         </dependency>
         <dependency>
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-lang3</artifactId>
-            <version>3.7</version>
+            <version>3.10</version>
```

Comment:
@maciejurbanskisaucelabs 
The projects sets [minimum JDK to 7](https://github.com/saucelabs/ci-sauce/blob/master/pom.xml#L84-L85), however `commons-lang3:3.10` [requires JDK 8](http://commons.apache.org/proper/commons-lang/release-notes/RELEASE-NOTES-3.10.txt):
> Lang 3.9 and onwards now targets Java 8, making use of features that arrived with Java 8.

Reply:
This might be a bit more of a challange to get it back to JDK 7 :( Please let us know if this is just a comment so that we update the project's minimum JDK or you require it to run with JDK 7.

Reply:
@adambabik no, we don't require JDK 7, I just would like to highlight that JDK version should be bumped (or if you have JDK 7 users, this change should be reverted)